skyportal/handlers/api/user.py

```diff
@@ -27,11 +28,87 @@ def get(self, user_id=None):
               application/json:
                 schema: Error
         """
-        user = User.query.options(joinedload(User.acls)).get(int(user_id))
+        user = User.query.get(int(user_id))
         if user is None:
             return self.error(f'Invalid user ID ({user_id}).')
         else:
-            return self.success(data=user)
+            user_info = user.to_dict()
+            user_info['acls'] = user.acls
+            return self.success(data=user_info)
+
+    @permissions(["Manage users"])
```

Comment:
In general, it seems like we should allow group admins to add users to the system. Super admins could add to any group, but we could restrict group admins only to add users to groups that that group admin has admin privileges to. As currently written, and as the default ACL, I worry that adding users to a Skyportal instance will be unnecessarily burdensome to the (few) super admins of a SkyPortal instance.

Reply:
Sure, I'll add "Manage users" to the group admin ACLs